src/utils/db.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'natali2023', {
```

Comment:
better to create constants
```suggestion
const DB_NAME = 'postgres';
// ...

const sequelize = new Sequelize(DB_NAME, 'postgres', 'natali2023', {
```